intel/filterlists/module.go

```diff
@@ -104,3 +102,11 @@ func stop() error {
 	filterListsLoaded = make(chan struct{})
 	return nil
 }
+
+func warnAboutDisabledFilterLists() {
+	module.Warning(
+		filterlistsDisabled,
+		"Filter Lists Are Initializing",
+		"The Filter Lists are waiting for update system to check for updates in order to retrieve the filter list data. Until the filter lists are fully initialized, the filter lists feature is disabled.",
+	)
```

Comment:
"Filter Lists" is used quite often here. Maybe change that to something like: "Filter lists are being downloaded and processed in the background. Until then the filter lists feature is disabled".